db/re/item_trade.txt

```diff
@@ -508,6 +508,14 @@
 5823,467,100	// Love_Arrow
 5855,499,100	// Fishing_Rod
 5864,475,100	// Shadow_Booster
+5900,2,100	// Divine Guard Hat
+5901,2,100	// Focus Beret
+5902,2,100	// Harvester Hat
+5903,2,100	// Deadman Bandana
+5904,2,100	// Inconspicuous Hat
+5905,2,100	// Lyrica Hat
+5906,2,100	// Oni Horns
+5907,2,100	// Sea Captain Hat
```

Comment:
I think the trade restriction should be 1+2+16+64+128+256=**467**
```
// 1   - item can't be dropped
// 2   - item can't be traded (nor vended)
// 16  - item can't be placed in the cart
// 64  - item can't be placed in the guild storage
// 128 - item can't be attached to mail
// 256 - item can't be auctioned
```

Reply:
Oh sorry, I'll fix now.